test.md

```diff
@@ -0,0 +1 @@
+This file should be ignored by git.
```

Comment:
- [ ] Please remove `test.md` from your repo because it's supposed to be ignored by git. I can see that you added it to the `.gitignore` file but this problem happened with you because you added it after you pushed the file already to Github.